internal/operators/mce/config.go

```diff
@@ -1,8 +1,61 @@
 package mce
 
+import (
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
+
+//go:embed ocp_mce_release_matrix.json
+var ocpMceReleaseMatrixFile []byte
+
+func getMCEVersion(openshiftVersion string) (*string, error) {
+
+	type OCPMCEReleaseMatrix struct {
+		OpenshiftVersion string   `json:"openshift_version"`
+		MceVersions      []string `json:"mce_versions"`
+	}
+
+	baseVersion, err := common.GetMajorMinorVersion(openshiftVersion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get base version of %s", openshiftVersion)
+	}
+
+	var matrix []OCPMCEReleaseMatrix
+	if err = json.Unmarshal(ocpMceReleaseMatrixFile, &matrix); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal ocp mce release matrix: %w", err)
+	}
+
+	var mceVersions []string
+	for _, record := range matrix {
+		if record.OpenshiftVersion == *baseVersion {
+			mceVersions = record.MceVersions
+		}
+	}
+	if len(mceVersions) == 0 {
+		return nil, errors.New("failed to find mce versions for the given openshift version")
+	}
+
+	latestVersion := mceVersions[0]
+	for _, version := range mceVersions {
+		isGreaterOrEqaul, err := common.VersionGreaterOrEqual(version, latestVersion)
+		if err != nil {
+			return nil, fmt.Errorf("failed to compare version %s with version %s", latestVersion, version)
+		}
+
+		if isGreaterOrEqaul {
+			latestVersion = version
```

Comment:
why keep a list of all the MCE versions for each OCP release if we just take the highest one?

Reply:
I thought it might be handy in case you wouldn't want the latest one in the future for some reason

Reply:
I think we should keep it simple, we can easily improve it later once we'll need it

Reply:
Ack.